PowerPointLabs/PowerPointLabs/RemoveHighlighting.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.Models;
+using Office = Microsoft.Office.Core;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs
+{
+    class RemoveHighlighting
+    {
+#pragma warning disable 0618
```

Comment:
new feature/codes should not disable 0618.
to do it in the right way, pls rewrite it using the Action framework for any new feature.
here's the ref: https://github.com/PowerPointLabs/PowerPointLabs/blob/master/doc/SoftwareDesign.md#to-create-a-new-feature